onnxruntime/python/tools/profile_explorer/profile_explorer.py

```diff
@@ -47,6 +50,28 @@ def _get_args():
     parser.add_argument("--csv", help="save data to csv")
     parser.add_argument("-c", "--count", type=int, default=40, help="list top N items")
     parser.add_argument("-v", "--verbose", action="store_true", help="verbose")
+
+    parser.add_argument(
+        "--start",
+        "-s",
+        type=int,
+        default=1,
+        help="index of the first model run to process",
+    )
+    parser.add_argument(
+        "--end",
+        "-e",
+        type=int,
+        default=-1,
+        help="index of the last model run to process (inclusive)",
```

Comment:
Please add a comment about what setting this to `-1` means, especially because that's the default value.